CHANGELOG.md

```diff
@@ -97,6 +104,21 @@ This changelog documents all notable changes to VAST and is updated on every rel
 - VAST now correctly builds within shallow clones of the repository. If the build system is unable to determine the correct version from `git-describe`, it now always falls back to the version of the last release.
   [#1570](https://github.com/tenzir/vast/pull/1570)
 
+- We fixed a regression that made it impossible to build static binaries from outside of the repository root directory.
+  [#1573](https://github.com/tenzir/vast/pull/1573)
+
+- The `VASTRegisterPlugin` CMake function now correctly removes the `ENTRYPOINT` from the given `SOURCES`, allowing for plugin developers to easily glob for sources again.
+  [#1573](https://github.com/tenzir/vast/pull/1573)
+
+- The `exporter.selectivity` metric is now 1.0 instead of NaN for idle periods.
+  [#1574](https://github.com/tenzir/vast/pull/1574)
+
+- VAST no longer renders JSON numbers with non-finite numbers as `NaN`, `-NaN`, `inf`, or `-inf`, resulting in invalid JSON output. Instead, such numbers are now rendered as `null`.
```

Comment:
This looks like a change, if not a breaking change. Not completely sure in which PR such changes should go.

Reply:
It was invalid JSON before, so this is most certainly a bug fix.

Reply:
>  in which PR such category changes should go

In this one, modified by hand.

But as I said above, this is a bug fix as the output was invalid JSON before.